README.md

```diff
@@ -445,6 +445,7 @@ Table of Contents
   * [parsers.dev](https://parsers.dev/) - Abstract syntax tree parsers and intermediate representation compilers as a service
   * [scan.coverity.com](https://scan.coverity.com/) — Static code analysis for Java, C/C++, C# and JavaScript, free for Open Source
   * [scrutinizer-ci.com](https://scrutinizer-ci.com/) — Continuous inspection platform, free for Open Source
+  * [Sema](https://www.semasoftware.com/) - Code review tool. Code reviewers have access to library of best practices to avoid repeating the same review.  Code reviews are automatically structured. There is also a dashboard for users to review the code and create portfolios of their work. Product is totally free to individuals and teams.
```

Comment:
doesnt appear to be something one can use now does it?

<img width="905" alt="Screenshot 2022-03-06 at 13 58 28" src="https://user-images.githubusercontent.com/82342/156924279-d436c671-4da3-4936-b312-615ac1c3d279.png">

Reply:
Hi @ripienaar users can use the product now. There is a waitlist to manage the flow of user onboarding, but we are letting in users same day at this point.

Reply:
Cool, well happy to add it once its unrestricted and open.